packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -63,6 +64,19 @@ export default class PciStoragesContainersAddController {
 
   onTypesFocus() {
     this.displaySelectedType = false;
+    this.loadingPrice = true;
+    this.PciProjectStorageContainersService.getPriceEstimation(
+      this.user.ovhSubsidiary,
+    )
+      .then((price) => {
+        this.price = price;
+        if (this.price !== null) {
+          this.price.formatedPrice = this.getPrice(price.price / 100000000);
```

Comment:
where does 10000000 comes from? is the price in micro cents?

Reply:
yes, it needs to be converted to show in Euros